docs/configuration.md

```diff
@@ -178,10 +191,31 @@ telemetry:
   enabled: true
 ```
 
+### Addons
+
+It is possible to bundle 3rd party helm charts with your cluster. 
+
+```
+helm:
+  repositories:
+  - name: stable
+    url: https://charts.helm.sh/stable
+  - name: prometheus-community
+    url: https://prometheus-community.github.io/helm-charts
+  charts:
+  - name: prometheus-stack
+    chartname: prometheus-community/prometheus
+    version: "11.16.8"
+    values: |2
+        values: "for overriding"
+    namespace: default
+```
+
```

Comment:
I think we should also document the CRD structure and how we pick those up from `kube-system` namespace.